react app/src/App.tsx

```
@@ -1,10 +1,250 @@
+      <section className="controls">
+        <label htmlFor="statementInput">Texto da fatura</label>
+        <textarea
+          id="statementInput"
+          value={statementText}
+          onChange={(event) => setStatementText(event.target.value)}
+          placeholder="Exemplo: 10/01 MERCADO CENTRAL 189,90"
+          rows={9}
+        />
+
+        <div className="actions">
+          <button onClick={handleParse}>Analisar fatura</button>
```

Biome (2.3.14)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)